src/main/java/org/jenkinsci/plugins/durabletask/AgentInfo.java

```diff
@@ -0,0 +1,169 @@
+package org.jenkinsci.plugins.durabletask;
```

Comment:
`AgentInfo` was just pulled out of `BourneShellScript` so that it could be used more generically by other classes.